StRoot/StMuDSTMaker/COMMON/StMuFstRawHit.cxx

```diff
@@ -0,0 +1,160 @@
+/***************************************************************************
+ *
+ * StMuFstRawHit.cxx
+ *
+ * Author: tchuang 2022
+ ***************************************************************************
+ *
+ * Description: Implementation of StMuFstRawHit, the StEvent hit structure
+ *
+ ***************************************************************************/
+#include "StMuFstRawHit.h"
+#include "St_base/StMessMgr.h"
+#include "StEvent/StEnumerations.h"
+#include "StRoot/StFstUtil/StFstRawHit.h"
+#include "StThreeVector.hh"
```

Comment:
... also I don't see a StThreeVector being used... again, we can remove this include...

Reply:
Done.